model/clusters_mgmt/v1/custom_iam_roles.model

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Contains the necessary attributes to support role-based authentication on AWS.
+struct CustomIAMRoles {
+	// Master IAMRole
+	MasterIAMRole String
+
+	// Worker IAMRole
```

Comment:
I would update these comments, as they show up in the API spec, to:
* The name of the IAM role that will be attached to master instances
* The name of the IAM role that will be attached to worker instances